commit ca6723004f060c7ba013866612d9ccb2b52ecf1b (refs/changes/96/2696/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-10-02T19:23:07+02:00 (2 years ago)
    
    TOOLS-1875 sdcadm update to node-cueball > v2.2.6
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 30d0a29..3436bc0 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "backoff": "2.5.0",
     "bunyan": "0.23.1",
     "cmdln": "4.2.1",
-    "cueball": "2.2.5",
+    "cueball": "2.4.0",
     "extsprintf": "^1.3.0",
     "kthxbai": "~0.4.0",
     "joyent-schemas": "git+https://github.com/joyent/schemas.git#385e6eb",
